packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -1452,6 +1452,35 @@ describe('TableView', function () {
         let before = tree.getByTestId('before');
         expect(document.activeElement).toBe(before);
       });
+
```

Comment:
Note: Wasn't able to write a test for the scroll position not changing when tabbing out/shift tabbing back into the Table. Tried a bunch of thing but couldn't get a test case that failed before the useSelectableCollection `focusWithoutScrolling` change